crates/biome_css_analyze/src/lint/nursery/no_unknown_unit.rs

```diff
@@ -0,0 +1,188 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic, RuleSource};
+use biome_console::markup;
+use biome_css_syntax::{
+    AnyCssDimension, CssFunction, CssGenericProperty, CssQueryFeaturePlain, CssSyntaxKind,
+};
+use biome_rowan::{SyntaxNodeCast, TextRange};
+
+use crate::utils::strip_vendor_prefix;
+
+const RESOLUTION_MEDIA_FEATURE_NAMES: [&str; 3] =
+    ["resolution", "min-resolution", "max-resolution"];
+
+fn is_css_hack_unit(value: &str) -> bool {
+    value == "\\0"
+}
+
+declare_rule! {
+    /// Disallow unknown units.
+    ///
+    /// This rule considers units defined in the CSS Specifications, up to and including Editor's Drafts, to be known.
```

Comment:
Is there a link to said specification that we can provide?

Reply:
I apologize for the oversight. I had directly copied the text from the stylelint's documentation ([link](https://stylelint.io/user-guide/rules/unit-no-unknown/)), which seems inappropriate for Biome documentation and lacks a specific link to the CSS specifications, leading to ambiguous explanations. I have now removed that text.

Reply:
We can use MDN link: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#lengths

Reply:
I have included the link you suggested in the documentation.